sql/src/test/java/io/crate/integrationtests/DDLIntegrationTest.java

```diff
@@ -366,6 +366,34 @@ public void test_alter_table_add_column_succedds_because_check_constaint_refers_
         execute("insert into t(id, qty, bazinga) values(0, 1, 42)");
     }
 
+    @Test
+    public void test_alter_table_drop_constraint_succeeds() {
```

Comment:
See previous comments. This is a bit vague. How is `succeeds` defined? 

```suggestion
    public void test_alter_table_drop_constraint_removes_the_constraint_and_leaves_other_constraints_in_place() {
```

Also makes it a bit more obvious that this test and `testAlterTableDropCheckConstraint` are kinda testing the same thing?

Reply:
yes, same test, so I have removed the least useful one, thank you